app/assets/stylesheets/calagator/index.scss

```diff
@@ -5,6 +5,7 @@
  * Then come the external dependencies:
  *
  *= require jquery.timepicker
+ *= require jquery-ui/theme
```

Comment:
Ditto here about the `*= require` syntax. Although, it appears we were already using it, so... :question: